From 687bd28ccac6005e8cfde4c70553468f69c6d9fc Mon Sep 17 00:00:00 2001
From: Tomeu Vizoso <tomeu.vizoso@collabora.com>
Date: Mon, 10 Dec 2018 09:18:29 +0100
Subject: [PATCH 143/151] panfrost: Don't emit framebuffer at context creation
 time

As at that moment no framebuffer has been set yet.

Signed-off-by: Tomeu Vizoso <tomeu.vizoso@collabora.com>
---
 src/gallium/drivers/panfrost/pan_context.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/gallium/drivers/panfrost/pan_context.c b/src/gallium/drivers/panfrost/pan_context.c
index fe50d19..f105283 100644
--- a/src/gallium/drivers/panfrost/pan_context.c
+++ b/src/gallium/drivers/panfrost/pan_context.c
@@ -2928,10 +2928,8 @@ panfrost_create_context(struct pipe_screen *screen, void *priv, unsigned flags)
         panfrost_emit_tiler_payload(ctx);
         panfrost_invalidate_frame(ctx);
         panfrost_viewport(ctx, 0.0, 1.0, 0, 0, ctx->pipe_framebuffer.width, ctx->pipe_framebuffer.height);
-        panfrost_new_frag_framebuffer(ctx);
         panfrost_default_shader_backend(ctx);
         panfrost_generate_space_filler_indices();
 
-
         return gallium;
 }
-- 
2.7.4

